src/Control Tasks/ACSControlTask.cpp

```diff
@@ -9,6 +9,7 @@ void ACSControlTask::execute()
     ACSWrite(constants::acs::xtorqorder, sfr::acs::current1, constants::acs::xout1, constants::acs::xout2, constants::acs::xPWMpin);
     ACSWrite(constants::acs::ytorqorder, sfr::acs::current2, constants::acs::yout1, constants::acs::yout2, constants::acs::yPWMpin);
     ACSWrite(constants::acs::ztorqorder, sfr::acs::current3, constants::acs::zout1, constants::acs::zout2, constants::acs::zPWMpin);
+    IMUOffset(sfr::temperature::temp_c, sfr::battery::voltage, sfr::acs::pwm1, sfr::acs::pwm2, sfr::acs::pwm3);
```

Comment:
Has this been re-confirmed with an actual IMU?

Reply:
Has what been confirmed? The dependence on those parameters? If so, not sure

Reply:
I misread the pull request originally and thought this included the fix for IMUMonitor hanging when the imu is disconnected,